### PR TITLE
add more examples

### PR DESCRIPTION
**Pull request by ms-henglu**

### CODE REVIEW

File: examples/Microsoft.Migrate/migrateProjects/main.tf [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+}
+
+provider "azapi" {
+  # Configuration options
+}
+
+provider "azurerm" {
+  features {
+
+  }
```
Comment (magodo) [inline examples/Microsoft.Migrate/migrateProjects/main.tf:16]: Can this be defined in a single line?

File: examples/Microsoft.Security/jitNetworkAccessPolicies/main.tf [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+}
+
+provider "azapi" {
+  # Configuration options
+}
+
+provider "azurerm" {
+  features {
+
+  }
```
Comment (magodo) [inline examples/Microsoft.Security/jitNetworkAccessPolicies/main.tf:16]: Same here.

File: examples/Microsoft.Security/jitNetworkAccessPolicies/main.tf [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+}
+
+# note: whilst these aren't used in all tests, it saves us redefining these everywhere
+locals {
+  first_public_key  = "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQC+wWK73dCr+jgQOAxNsHAnNNNMEMWOHYEccp6wJm2gotpr9katuF/ZAdou5AaW1C61slRkHRkpRRX9FA9CYBiitZgvCCz+3nWNN7l/Up54Zps/pHWGZLHNJZRYyAB6j5yVLMVHIHriY49d/GZTZVNB8GoJv9Gakwc/fuEZYYl4YDFiGMBP///TzlI4jhiJzjKnEvqPFki5p2ZRJqcbCiF4pJrxUQR/RXqVFQdbRLZgYfJ8xGB878RENq3yQ39d8dVOkq4edbkzwcUmwwwkYVPIoDGsYLaRHnG+To7FvMeyO7xDVQkMKzopTQV8AuKpyvpqu0a9pWOMaiCyDytO7GGN you@me.com"
+  second_public_key = "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQC0/NDMj2wG6bSa6jbn6E3LYlUsYiWMp1CQ2sGAijPALW6OrSu30lz7nKpoh8Qdw7/A4nAJgweI5Oiiw5/BOaGENM70Go+VM8LQMSxJ4S7/8MIJEZQp5HcJZ7XDTcEwruknrd8mllEfGyFzPvJOx6QAQocFhXBW6+AlhM3gn/dvV5vdrO8ihjET2GoDUqXPYC57ZuY+/Fz6W3KV8V97BvNUhpY5yQrP5VpnyvvXNFQtzDfClTvZFPuoHQi3/KYPi6O0FSD74vo8JOBZZY09boInPejkm9fvHQqfh0bnN7B6XJoUwC1Qprrx+XIy7ust5AEn5XL7d4lOvcR14MxDDKEp you@me.com"
+}
```
Comment (magodo) [inline examples/Microsoft.Security/jitNetworkAccessPolicies/main.tf:23]: y Can this be some redacted string like `"REDACTED"`? As this is only an example, users are always expected to replace it with their own public key?

Also the `second_public_key` seems not used anywhere?

File: examples/Microsoft.Security/jitNetworkAccessPolicies/main.tf [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+    offer     = "UbuntuServer"
+    sku       = "16.04-LTS"
+    version   = "latest"
+  }
+}
+
+resource "azapi_resource" "test" {
+  type      = "Microsoft.Security/locations/jitNetworkAccessPolicies@2020-01-01"
+  name      = "myPolicy"
+  parent_id = "${azurerm_resource_group.test.id}/providers/Microsoft.Security/locations/westeurope"
```
Comment (magodo) [inline examples/Microsoft.Security/jitNetworkAccessPolicies/main.tf:87]: Is this also correctly parsed? Should you also mention this in the document about resource identifier?

File: examples/Microsoft.Insights/workbooks/main.tf [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+  type      = "Microsoft.Insights/workbooks@2021-08-01"
+  name      = "deadb33f-8bee-4d3b-a059-9be8dac93960"
+  parent_id = azurerm_resource_group.test.id
+  location  = azurerm_resource_group.test.location
+  body = jsonencode({
+    kind = "user"
+    properties = {
+      category       = "workbook"
+      displayName    = "My Workbook"
+      serializedData = "{\"version\":\"Notebook/1.0\",\"items\":[{\"type\":1,\"content\":\"{\"json\":\"## New workbook\\r\\n---\\r\\n\\r\\nWelcome to your new workbook.  This area will display text formatted as markdown.\\r\\n\\r\\n\\r\\nWe've included a basic analytics query to get you started. Use the `Edit` button below each section to configure it or add more sections.\"}\",\"halfWidth\":null,\"conditionalVisibility\":null},{\"type\":3,\"content\":\"{\"version\":\"KqlItem/1.0\",\"query\":\"union withsource=TableName *\\n| summarize Count=count() by TableName\\n| render barchart\",\"showQuery\":false,\"size\":1,\"aggregation\":0,\"showAnnotations\":false}\",\"halfWidth\":null,\"conditionalVisibility\":null}],\"isLocked\":false}"
```
Comment (magodo) [inline examples/Microsoft.Insights/workbooks/main.tf:58]: Can this be heredoc for readability?

File: examples/Microsoft.Insights/workbooks/main.tf [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+    }
+  }
+}
+
+provider "azurerm" {
+  features {}
+}
+
+provider "azapi" {
+}
```
Comment (magodo) [inline examples/Microsoft.Insights/workbooks/main.tf:14]: Can we be consistent about this among the example cfgs?

**Review by magodo — CHANGES_REQUESTED** [review]
Thanks for this PR!
I have left some additional mostly minor comments that once addressed this should be good to merge 👍

File: examples/Microsoft.Insights/workbooks/main.tf (hunk 5 above)
Comment (ms-henglu) [reply examples/Microsoft.Insights/workbooks/main.tf:58]: It has another encoded json string inside, so I prefer to keep it as it is in the example.

**Comment by ms-henglu** [comment]
Hi @magodo ,

Thanks for code review, I've updated the PR, would you please take another look? Thanks!

**Review by magodo — APPROVED** [review]

**Pull request merged**